Dockerfile

```diff
@@ -1,16 +1,12 @@
 # Base node image
-FROM node:19-alpine AS node
+FROM node:20-alpine AS node
 
 COPY . /app
 WORKDIR /app
 
 # Install call deps - Install curl for health check
 RUN apk --no-cache add curl && \
-    # We want to inherit env from the container, not the file
-    # This will preserve any existing env file if it's already in source
-    # otherwise it will create a new one
-    touch .env && \
```

Comment:
In fact I only realized now why this might in fact be useful:

when running `docker-compose up` without a `.env` file, `docker-compose` creates a _directory_ called `.env`. This might confuse users (at least it would confuse me) when they want to add the `.env` file later.

So it might be useful to keep this `touch .env` here, but adjust the comment accordingly.

Reply:
In fact that doesn't work. I just tried it, and if the `.env` file doesn't exist in the user directory, docker will create a directory anyway, even if the docker-image has a `.env` file!